src/main/thrift/parquet.thrift

```diff
@@ -626,11 +628,9 @@ struct FileMetaData {
   6: optional string created_by
 
   /**
-   * Sort order used for each column in this file.
-   *
-   * If this list is not present, then the order for each column is assumed to
```

Comment:
If the column order for a column is not set, Impala only reads stats for numeric types ([parquet-column-stats.cc#L130](https://github.com/apache/incubator-impala/blob/master/be/src/exec/parquet-column-stats.cc#L130)). We treat the values as having been written by older versions of parquet-mr, thus only using stats for types that did not have known issues in the past.

```
  if (col_order == nullptr) {
    return col_type.IsBooleanType() || col_type.IsIntegerType()
        || col_type.IsFloatingPointType();
  }
```